src/dev/display/st7735/lv_st7735.c

```diff
@@ -0,0 +1,170 @@
+/**
+ * @file lv_st7735.c
+ *
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "lv_st7735.h"
+
+/*********************
+ *      DEFINES
+ *********************/
+
+#define CMD_GAMSET		0x26
+
+#define CMD_FRMCTR1		0xB1
+#define CMD_FRMCTR2		0xB2
+#define CMD_FRMCTR3		0xB3
+#define CMD_INVCTR		0xB4
+#define CMD_DISSET5		0xB6
+
+#define CMD_PWCTR1		0xC0
+#define CMD_PWCTR2		0xC1
+#define CMD_PWCTR3		0xC2
+#define CMD_PWCTR4		0xC3
+#define CMD_PWCTR5		0xC4
+#define CMD_VMCTR1		0xC5
+#define CMD_VMOFCTR		0xC7
+
+#define CMD_NVFCTR1		0xD9
+
+#define CMD_GMCTRP1		0xE0
+#define CMD_GMCTRN1		0xE1
+
+#define CMD_PWCTR6		0xFC
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/**********************
+ *  STATIC PROTOTYPES
+ **********************/
+
+static void send_cmd(lv_lcd_generic_mipi_driver_t *drv, uint8_t cmd, uint8_t *param, size_t param_size);
+static lv_lcd_generic_mipi_driver_t *get_driver(lv_display_t *disp);
+
+/**********************
+ *  STATIC CONSTANTS
+ **********************/
+
+static const uint8_t init_cmd_list[] = {
+#if 0
```

Comment:
`#if 0`s should be removed and only the final code should remain.
If the other options might be used by the users as well, and API functions or other direct way to adjust shall be provided.